src/MyPlot/EventListener.php

```diff
@@ -234,56 +234,57 @@ public function onEntityMotion(EntityMotionEvent $event) {
 	 * @param PlayerMoveEvent $event
 	 */
 	public function onPlayerMove(PlayerMoveEvent $event) {
-		if($event->isCancelled()) {
+		if($event->isCancelled() or $event->getFrom()->distance($event->getTo()) < 0.01) {
```

Comment:
Nice. Now you revertet the performance to shitty because rotation now is not ignored anymore. Fuck this, i start my own fork

Reply:
Players have the ability to move less than a block. With your previous code, a player could have moved 0.0000001 into the next block and have been called for movement. This resolved that because it used distance rather than floored vectors

Reply:
Do you even understand your code? I guess not. guess what floor() is for?